src/admin/containers/AddonPage/index.js

```diff
@@ -33,19 +33,22 @@ class AddonPage extends React.Component {
     ];
     if (addon.homepage) {
       items.push([
+        // eslint-disable-next-line react/jsx-key
```

Comment:
It's actually using data in the object to make the key later on. The eslint rule just sees any jsx in an array as needing a key so this is a false positive.